pkg/services/accesscontrol/ossaccesscontrol/permissions_services.go

```diff
@@ -176,7 +176,15 @@ type FolderPermissionsService struct {
 }
 
 var FolderViewActions = []string{dashboards.ActionFoldersRead}
-var FolderEditActions = append(FolderViewActions, []string{dashboards.ActionFoldersWrite, dashboards.ActionFoldersDelete, dashboards.ActionDashboardsCreate}...)
+var FolderEditActions = append(FolderViewActions, []string{
+	dashboards.ActionFoldersWrite,
+	dashboards.ActionFoldersDelete,
+	dashboards.ActionDashboardsCreate,
+	accesscontrol.ActionAlertingRuleRead,
```

Comment:
shouldn't it be part of FolderViewActions?

Reply:
Trying to figure this out. I think for non RBAC a user with "View" permission on a folder can read alert rules in that folder, user with "Edit" or "Admin" can read, create, update and delete alert rules?

Reply:
Yes you are correct